internal/state/change_processor_test.go

```diff
@@ -2118,8 +2100,7 @@ var _ = Describe("ChangeProcessor", func() {
 							},
 						},
 					},
-					SSLServers:    []dataplane.VirtualServer{},
-					BackendGroups: []graph.BackendGroup{bg},
+					SSLServers: []dataplane.VirtualServer{},
```

Comment:
There is a related comment to that - it is a bit confusing why we have zero backend groups here in the Configuration.

Reply:
Addressed by allowing BackendGroups with zero backends.